OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -429,6 +430,22 @@ public boolean areNotificationsEnabled() {
         }
     }
 
+    public static class YoutubeConfig {
```

Comment:
Same question as on the iOS PR, really the change this config enables is EMBEDDED YouTube. Youtube is supported in the apps already, just not embedded youtube. It's probably ok as is.

Reply:
ok